app-network/netbird/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=netbird
+PKGDES="Mesh VPN based on WireGuard"
+BUILDDEP="go git"
+PKGDEP="glibc"
+PKGSEC=net
+
+ABTYPE=self
+ABSPLITDBG=0
```

Comment:
Drop this line, Autobuild is now able to handle debug symbol stripping from Go applications.